activity.py

```diff
@@ -21,7 +21,8 @@ def __init__(self, handle):
 
         # Create the game instance.
         self.game = Countries.Countries()
-        self.game.canvas = sugargame.canvas.PygameCanvas(self, main=self.game.run,
+        self.game.canvas = sugargame.canvas.PygameCanvas(
+                                                         self, main=self.game.run,
                                                          modules=[pygame.display,
                                                                   pygame.font,
                                                                   pygame.mixer])
```

Comment:
This looks a bit better, what'd you think?
```python3
self.game.canvas = sugargame.canvas.PygameCanvas(
    self,
    main=self.game.run,
    modules=[pygame.display,
             pygame.font,
             pygame.mixer]
)
```

Reply:
Yes this looks much better, I will make this change